[Rewrite] Function Wikis - #4 (#213)
NilPointer-Software/bdfd-wiki: src/bdscript/suppressErrors.md

# $suppressErrors

> Prevents the default function error messages from being send whenever an error occurs.

Reviewer: ```suggestion
Prevents the default function error message from being sent whenever an error occurs.
```

## Usages

> There are two usages of this function :

Reviewer 2: ```suggestion
There are two usages of this function:
```

### Usage #1
#### Syntax
```
$suppressErrors
```

> Prevents the default error message from being send.

Reviewer: ```suggestion
Prevents the default error message from being sent.
```

#### Example
```
$nomention
$suppressErrors
$argsCheck[>1;Usage : ` !command <math expression> `]
$calculate[$message]
```
![Screenshot_20221029_133343](https://user-images.githubusercontent.com/95774950/198821156-3752e153-2289-4cd8-8bfb-07ae72455530.png)

### Usage #2
#### Syntax
```
$suppressErrors[Error message]
```
Sends a custom error message instead of the default error message.

#### Parameters

> - `Error message` : The custom error message to send.

Reviewer 2: ```suggestion
- `Error message`: The custom error message to send.
```
